hazelcast/src/main/java/com/hazelcast/spi/OperationResponseHandler.java

```diff
@@ -29,6 +29,9 @@
      *
      * @param op       the operation that got executed.
      * @param response the response of the operation that got executed.
+     * @throws com.hazelcast.nio.serialization.HazelcastSerializationException if response is not serializable or
```

Comment:
should we import `com.hazelcast.nio.serialization.HazelcastSerializationException` so that you can just write `HazelcastSerializationException` here.